include/mata/afa.hh

```diff
@@ -43,7 +44,7 @@ extern const std::string TYPE_AFA;
 // START OF THE DECLARATIONS
 
 using State = Mata::Nfa::State;
-using Symbol = Mata::Nfa::Symbol;
+using Symbol = Mata::Symbol;
```

Comment:
We cannot use these using directive renames for existing types in header files. This would introduce new types in different namespaces and would clutter the code.

We have to discuss this further in the meeting on Thursday.